tests/core/util/test_file_keyring_synchronization.py

```diff
@@ -62,13 +61,14 @@ def dummy_set_passphrase(service, user, passphrase, keyring_path, index, num_wor
 
 class TestFileKeyringSynchronization:
     # When: using a new empty keyring
+    @pytest.mark.benchmark
```

Comment:
Let's make sure we talk through this before tossing stuff on the benchmark runner for reasons other than them being tests verifying performance.

Reply:
Im not really intending to merge it like this, its still a draft :)